src/config.ts

```diff
@@ -366,6 +377,25 @@ function normalizeScripts(cwd: string, scripts: RawScripts): BuildScript[] {
   return processedScripts;
 }
 
+type RawProxies = Record<string, string | ProxyOptions>;
+function normalizeProxies(proxies: RawProxies) {
+  return Object.entries(proxies).reduce<Proxy[]>((acc, [context, options]) => {
+    if (typeof options !== 'string') return [...acc, [context, options]];
+    return [
+      ...acc,
+      [
+        context,
+        {
+          target: options,
+          ignorePath: true,
+          secure: false,
+          changeOrigin: true,
+        },
```

Comment:
Now that I check the codebase, I think moving default values to `config.ts` makes sense, good job @stramel 👌

Reply:
Thanks 😄

Reply:
@thisguychris I was curious why you added `secure: false`. By default it only does secure if you use a `https` target. With the `changeOrigin: true` flag, it shouldn't need the secure flag set, I think. 🤔

Reply:
makes sense to use the default behavior, will remove when I merge unless I hear otherwise

Reply:
@stramel @FredKSchott 
hey guys 👋 sorry for commenting late.

I don't think the default value is false checking the README, it just says you can set true or false. So it means you have to be explicit when setting?

Diving to http-proxy code, you can see, that if you set it undefined (did not set like here), `[rejectUnauthorized flag is actually set to true](https://github.com/http-party/node-http-proxy/blob/master/lib/http-proxy/common.js#L58)?

Also a lot of articles and their issue list points to add [secure false](https://sdk.gooddata.com/gooddata-ui/docs/cors.html)

Which enables you to use self signed certificate during production which is a lot more common on working with the dev environment.

> By default it only does secure if you use a https target.

So to answer this, removing secure is [not equal](https://github.com/http-party/node-http-proxy/blob/master/lib/http-proxy/common.js#L58) to having explicitly saying `secure: false`